app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="groups-index">
```

Comment:
### [OPTIONAL]
- Awesome work on implementing the mobile design but it would look much better if you made it responsive to the desktop version as well. Currently, it looks like that in the screenshot below and it doesn't look appealing.
![image](https://github.com/BANCUNGUYE66/budget-app/assets/45460192/0c30ed36-d1f1-4091-9170-a2464d5ac39a)